services/graphql-server/src/graphql/definitions/email/campaign.js

```diff
@@ -1,11 +1,80 @@
 const gql = require('graphql-tag');
 
 module.exports = gql`
+  extend type Query {
```

Comment:
Fix the indentation to match the rest of the GraphQL definitions, there should be a line break before the start of the definition, and  the root level should start without any indentation (rather than the 2 spaces here).